hazelcast/src/main/java/com/hazelcast/instance/DefaultNodeExtension.java

```diff
@@ -156,7 +156,7 @@ public void printNodeInfo() {
         systemLogger.info("Hazelcast " + buildInfo.getVersion()
                 + " (" + build + ") starting at " + node.getThisAddress());
         systemLogger.info("Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.");
-        systemLogger.info("Configured Hazelcast Serialization version: " + buildInfo.getSerializationVersion());
+        systemLogger.fine("Configured Hazelcast Serialization version: " + buildInfo.getSerializationVersion());
```

Comment:
we only have 1 serialization version and it's extremely unlikely to change before Hazelcast 4